### PR TITLE
Content updates for connection banner

### LINKED ISSUE #5623: Connection banner: review copy
> Let’s take a step back and go through each of those panels and spend some time on the copy.
> [...]
> The last step feels like it should be some sort of “let’s do this, get set up now” kind of thing not a “and more… stuff” panel.

#5473

p2JiWH-1tm-p2

cc @richardmuscat

### PR DESCRIPTION
**Pull request by richardmuscat**
Fixes #5623

### CODE REVIEW

**Comment by jeherve** [comment]
Will also fix #5622

File: class.jetpack-connection-banner.php [hunk 1]
```diff
@@ -230,7 +230,7 @@ class="notice-dismiss"
 
 					<!-- slide 1: intro -->
 					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-one jp__slide-is-active">
-						<h2><?php esc_html_e( 'Jetpack is here to enhance your site and your life', 'jetpack' ) ?></h2>
+						<h2><?php esc_html_e( 'Jetpack simplifies site security, customization, and promotion.', 'jetpack' ) ?></h2>
```
Comment (jeffgolenski) [inline class.jetpack-connection-banner.php]: Promotion? What does that mean to an end user? It sounds like Jetpack is a marketing tool.

File: class.jetpack-connection-banner.php [hunk 2]
```diff
@@ -246,9 +246,25 @@ class="notice-dismiss"
 						<p>
 							<?php
 							esc_html_e(
-								'Jetpack is a free plugin that utilizes powerful WordPress.com servers to simplify 
-									managing your site(s) with detailed visitor stats, security services, 
-									image performance upgrades, and more. Connect with WordPress.com to get started!',
+								'Jetpack is a free plugin that utilizes powerful WordPress.com servers to augment your site and simplify managing it.',
```
Comment (jeffgolenski) [inline class.jetpack-connection-banner.php]: I don't anyone is going to know what "augment your site" means

File: class.jetpack-connection-banner.php [hunk 3]
```diff
@@ -382,15 +429,19 @@ class="dops-button is-primary">
...
+								'Publish content, track stats, moderate comments and so much more from anywhere in the world. Our mobile and desktop apps are free and available to you on Apple or Android devices once Jetpack is connected to WordPress.com.',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p>
+							<?php
+							esc_html_e(
+								'When Jetpack is connected to WordPress.com, head over to the Apps tab within Jetpack for direct links to the mobile and desktop apps.',
```
Comment (jeffgolenski) [inline class.jetpack-connection-banner.php]: this is great!

File: class.jetpack-connection-banner.php (hunk 1 above)
Comment (richardmuscat) [inline class.jetpack-connection-banner.php]: Means promoting your website. Traffic tools (publicize, sharing, enh distribution, subscriptions, etc).

Less vague than "enhance your life" I thought :)

Comment (richardmuscat) [inline class.jetpack-connection-banner.php]: Fixed

File: class.jetpack-connection-banner.php (hunk 2 above)
Comment (richardmuscat) [inline class.jetpack-connection-banner.php]: Fixed

**Comment by dereksmart** [comment]
👍

File: class.jetpack-connection-banner.php (hunk 1 above)
Comment (jeffgolenski) [inline class.jetpack-connection-banner.php]: @richardmuscat touchè! I'm just playing devil's advocate here. It's also worth mentioning that I pulled a lot of that text straight from the wordpress.org plugin repo. We should consider keeping them consistent. I'd love to see a lot of these additions you've made transfer over there!

**Comment by zinigor** [comment]
Ported to `branch-4.4` in 0438997.

**Pull request merged**